services/jaeger/2.29.0/pre-upgrade/delete-jaeger-deployment.yaml

```diff
@@ -51,4 +51,4 @@ spec:
             - sh
             - -c
             - |
-              kubectl delete deployment jaeger-jaeger-operator -n istio-system --ignore-not-found
+              kubectl delete deployments.apps -l app.kubernetes.io/name=jaeger-operator,app.kubernetes.io/instance!=jaeger-operator --cascade=orphan -n istio-system --ignore-not-found
```

Comment:
is this label present in older version of jaeger and NOT in newer version of jaeger?
can we add a comment on how/where this label comes from? (Thinking of the next jaeger bump in future).

Reply:
https://github.com/jaegertracing/helm-charts/pull/257/files#diff-d9ac456c418c74b7ba25a05ec2cc077cd8dd45ddb7a3008c2dee25e4d347a23eR48 `app.kubernetes.io/instance` label was added in newer version, so selecting against here. I'll add a comment (I was thinking we would probably be removing - or updating - 2.2 upgrade strategies in the next 2.3 release?)

Reply:
> I was thinking we would probably be removing - or updating - 2.2 upgrade strategies in the next 2.3 release?)

:crossed_fingers: